internal/ingress/status/status.go

```diff
@@ -185,7 +185,7 @@ func NewStatusSyncer(config Config) Sync {
 			go st.syncQueue.Run(time.Second, stop)
 			wait.PollUntil(updateInterval, func() (bool, error) {
 				// send a dummy object to the queue to force a sync
-				st.syncQueue.Enqueue("sync status")
```

Comment:
why do we have to reload/regenerate Nginx config when controller leader change?

Reply:
This is unnecessary now. Removed.